server/src/main/scala/za/co/absa/atum/server/api/database/DoobieImplicits.scala

```diff
@@ -155,7 +155,29 @@ object DoobieImplicits {
           }
         )
     }
+  }
 
+  implicit val encodeResultValueType: Encoder[MeasureResultDTO.ResultValueType] = Encoder.encodeString.contramap {
```

Comment:
What's the motivation to place json related encoders/decoders alongside doobie implicits?

Reply:
Not sure I understand the question, those JSON related SerDe code was already there & I needed those MeasureResult DTOs to be serialized/deserialized as well

Reply:
DoobieImplicits object is there for defining Put/Get/Read/Write instances for Doobie. Play json implicits are supposed to be somewhere else - PlayJsonImplicits object in model package.

Reply:
Aaah. Yes, I'm sorry I understand now. I'll move them to `CirceImplicits.scala`

I know that I could move them directly to `CheckpointFromDB.scala`, but I anticipate that @TebaleloS will create a bunch of them later as well, so it might be a good idea for them to be centralized

Reply:
It's customary to place them in companion objects.

Reply:
moved into companion object, thanks for the recommendation